Gemfile

```diff
@@ -1,6 +1,6 @@
 source 'https://rubygems.org'
 
-ruby '3.0.2'
+ruby '3.0.0'
```

Comment:
This looks like we've downgraded our Ruby version - is there a specific reason for that?